### ISSUE TITLE
Tinker with 1/2-wave aerial

### ISSUE THREAD

**Issue by JackKelly** [JackKelly/rfm_edf_ecomanager#33]
Turn on just a single TX.

Count number of received packets in 10 minutes.

Add 1/2 wave aerial.  Count again.

**Comment by JackKelly** [comment]
Lengths: http://blog.strobotics.com.au/2008/06/17/rfm12-tutorial-part2/

**Comment by JackKelly** [comment]
Some results from experiments:

quarter wave with no ground plane = 8035.488 seconds / 961 packets = 8.362 seconds / sample

quarter wave with ground plane = 73058.826 seconds / 10769 packets = 6.784 seconds / sample

**Comment by JackKelly** [comment]
quarter wave with ground plane after tweaks done on Friday 26th Oct = 7.146 seconds / sample

**Comment by JackKelly** [comment]
# Quarter wave with ground plane and tweaks done end of Fri 26th Oct

## Keep offset independent of VDI:

### Polling one TRX and receiving one TX:

```
TX
Mean num secs per sample:  6.74s
Max secs between samples: 30.94s
Min secs between samples:  5.87s
stdev between samples   :  2.03s

```

### Just receiving one TX:

```
TX
Mean num secs per sample:  6.64s
Max secs between samples: 24.79s
Min secs between samples:  5.87s
stdev between samples   :  1.88s
```

## Offset linked to VDI:

### Polling one TRX and receiving one TX:

```
TX
Mean num secs per sample:  6.79s
Max secs between samples: 29.65s
Min secs between samples:  5.87s
stdev between samples   :  2.11s

TRX
Mean num secs per sample:  6.00s
Max secs between samples:  6.30s
Min secs between samples:  5.80s
stdev between samples   :  0.02s
```

### Just receiving one TX:

```
TX
Mean num secs per sample:  6.64s
Max secs between samples: 30.56s
Min secs between samples:  5.86s
stdev between samples   :  1.82s
```

# Half wave with ground plane and tweaks done end of Fri 26th Oct

## Offset linked to VDI:

### Just receiving one TX:

```
TX
Mean num secs per sample:  6.72s
Max secs between samples: 24.52s
Min secs between samples:  5.90s
stdev between samples   :  1.96s
```

# Half wave without ground plane and tweaks done end of Fri 26th Oct

## Offset linked to VDI:

### Just receiving one TX:

```
Duration = 250208.72s

TX
Mean num secs per sample:  6.70s
Max secs between samples: 30.90s
Min secs between samples:  5.85s
stdev between samples   :  1.91s
```

**Comment by JackKelly** [comment]
None of the setups listed above appear to offer significantly different performance.  So half wave or quarter wave with ground plane should be fine.

**Comment by JackKelly** [comment]
It seems the aerial config does have a significant affect on how well we can poll TRXs the other side of the house.

Half wave aerial WITH ground plane works significantly better than half wave WITHOUT gp.

Still to try: quarter wave with and without gp.

**Comment by JackKelly** [comment]
must also update the documentation if there's a clear winner for aerial config.  Early results suggest quarter wave with ground plain is a clear winner.

**Comment by JackKelly** [comment]
I've also tinkered with a dipole (where at 1/4-wave wire runs upwards from the RFM12b's antenna connection and a 1/4-wave wire runs downwards from the Nanode's ground (and the two wires are aligned so, from a distance, they look like a single straight 1/2-wave length wire with a Nanode in the middle).  The dipole appears to work about as well as the 1/4-wave with ground plane, except for receiving my EDF whole-house TX for some reason.

The bottom line is that either a 1/4-wave with ground plane (with radius >= 1/4-wavelength) or a dipole appears to work well.  There's some good discussion on [the sparkfun comments thread for the RFM12b](https://www.sparkfun.com/products/9582#comment-4eaad858757b7fd35100b2dd).
